src/PKSim.Presentation/DTO/ExpressionProfiles/ExpressionProfileDTO.cs

```diff
@@ -48,7 +48,7 @@ public virtual string MoleculeName
 
       private bool moleculeSpeciesCategoryValid(string name) => !_allExistingExpressionProfileNames.Contains(name?.TrimmedValue()?.ToLower());
 
-      public string Name => ExpressionProfileName(MoleculeName, Species, Category);
+      public string Name => Constants.ContainerName.ExpressionProfileName(MoleculeName, Species?.DisplayName, Category);
```

Comment:
ExpressionProfileName. Isn't this method also in core?

Reply:
@msevestre  this is the on :D in Constants.ContainerName